### PR TITLE
CODE RUB: Updated Variable in retrieve all Calendar Entry Attachments

### PR DESCRIPTION
**Pull request by deepshri7**

### CODE REVIEW

**Comment by hassanhabib** [comment]
@deepshri7 fix conflicts here please

**Comment by deepshri7** [comment]
No conflicts in this.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 1]
```diff
@@ -26,12 +26,12 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllCalendarEntryAttachmentsW
                     .Throws(sqlException);
 
             // when
-            Action retrieveAllCalenderEntryAttachmentAction = () =>
-                this.calendarEntryAttachmentService.RetrieveAllCalendarEntryAttachments();
+            Action retrieveAllCalendarEntryAttachmensAction = () =>
+            this.calendarEntryAttachmentService.RetrieveAllCalendarEntryAttachments();
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs:30]: Tab is required

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 2]
```diff
@@ -26,12 +26,12 @@ public void ShouldThrowDependencyExceptionOnRetrieveAllCalendarEntryAttachmentsW
...
 
             // then
             Assert.Throws<CalendarEntryAttachmentDependencyException>(
-                retrieveAllCalenderEntryAttachmentAction);
+              retrieveAllCalendarEntryAttachmensAction);
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs:34]: check the alignment here

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs [hunk 3]
```diff
@@ -60,13 +60,13 @@ public void ShouldThrowServiceExceptionOnRetrieveAllCalendarEntryAttachmentsWhen
                 broker.SelectAllCalendarEntryAttachments())
                     .Throws(serviceException);
 
-            // when
-            Action retrieveAllCalenderEntryAttachmentAction = () =>
+             // when
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/CalendarEntryAttachments/CalendarEntryAttachmentServiceTests.Exceptions.RetrieveAll.cs:63]: remove this extra space

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Pull request closed without merging**